__tests__/cron/personalizedDigest.ts

```diff
@@ -81,7 +83,28 @@ describe('personalizedDigest cron', () => {
       expect(notifyGeneratePersonalizedDigest).toHaveBeenCalledWith(
         expect.anything(),
         personalizedDigest,
+        expect.any(Number),
       );
     });
   });
+
+  it('should log notify count', async () => {
```

Comment:
Usually I don't recommend testing for logs but up to you :)

Reply:
I can see what you mean. From my point of view it's a critical part of the feature since we want to make sure logs are there to know how many digests were processed. In the future that could ping some other service (eg. tracking) instead of logging and we would want to test that?